department-of-veterans-affairs/caseflow
# Correspondence and Appeal Tasks: Can view Veteran documents by selecting “view documents”

Resolves [Correspondence and Appeal Tasks: Can view Veteran documents by selecting “view documents”](https://jira.devops.va.gov/browse/APPEALS-42527?filter=-1)

# Description
This ticket creates a View veteran documents link on the existing appeals section of the page. When clicked it will take the user to the documents page.

## Acceptance Criteria

## Testing Plan
1. Acquire a link to the correspondence details page, should look something like :
/queue/correspondence/8d0b2b6f-a5b9-4893-80bd-4122df8bb2d9
2. Login as someone who has access to reader (BVADWISE will work)
3. Paste the URL in, and click the view veteran documents
4. Confirm that you reached the reader URL (veteran might have no documents present in reader)

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1219" alt="image" src="https://github.com/user-attachments/assets/5dd53fc6-87b4-47b8-b312-bb3fd25df8be">
|<img width="1239" alt="image" src="https://github.com/user-attachments/assets/e317136b-b219-4dfb-952c-98c1618de3ed">

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
